packages/slate-editor/src/modules/editor-v4-image/ImageExtension.tsx

```diff
@@ -100,15 +99,6 @@ const ImageExtension = ({
                 event.stopPropagation();
                 return;
             }
-
-            if (isDeletingBackward(event) && EditorCommands.isSelectionAtBlockStart(editor)) {
-                EditorCommands.removeNode(editor, {
-                    at: nodeEntry[1],
-                    match: isImageNode,
-                });
-                event.preventDefault();
-                event.stopPropagation();
-            }
```

Comment:
Does this mean it's not possible to delete the block by pressing backspace?

Scenario:

1. I have an image block
2. I have a text with paragraph after it
3. I put my cursor to the end of the paragraph
4. I press Backspace multiple types
5. I expect the paragraph contents to be deleted one character after another
6. Once the paragraph is gone I expect my cursor to:
   - Either remove the image block and make the document empty
   - Or move to the image caption text and start deleting it. 
     Once the caption is empty, my next Backspace pressing should remove the image block

How this scenario will work after this change?

Reply:
@e1himself when the caption is empty and you press Backspace the image block will be removed and the selection will go to the block bellow
![2021-12-16 17 07 47](https://user-images.githubusercontent.com/3620639/146387142-906bc121-999f-41e0-8b2e-4d2496bde45d.gif)

Reply:
Good! 

What about step 6 from the scenario above? Will it start deleting the caption or will it remove the image right away if I continue pressing Backspace after deleting the paragraph following the image block?

Reply:
hm, i will take a look

Reply:
@aspirisen so what will be the behavior now?

Reply:
@e1himself 
![2021-12-16 18 13 17](https://user-images.githubusercontent.com/3620639/146397957-3488fac1-0c0d-41e8-9368-2c0633cdbd1a.gif)

Reply:
For future reference: we'll follow up on improving the behavior with backspace and void elements in MT-4532 and MT-4533.